src/Adapters/Storage/Oracle/OracleStorageAdapter.js

```
@@ -0,0 +1,1156 @@
+      index += 1;
+      continue;
+    } else if (typeof fieldValue === 'string') {
+      patterns.push(`$${index}:name = $${index + 1}`);
+      values.push(fieldName, fieldValue);
+      index += 2;
+    } else if (typeof fieldValue === 'boolean') {
+      patterns.push(`$${index}:name = $${index + 1}`);
+      // Oracle uses NUMBER(1) for boolean
+      if (schema.fields[fieldName] && schema.fields[fieldName].type === 'Number') {
+        // Should always return zero results
+        const MAX_INT_PLUS_ONE = 9223372036854775808;
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 9223372036854776000

(lint/correctness/noPrecisionLoss)